docs/imagemanifestv2.md

```diff
@@ -0,0 +1,113 @@
+# Image Manifest V2 
+
+This document outlines the format of of the V2 image manifest.  Image manifests describe the various constituents of a docker image.  Image manifests can be serialized to JSON format.
+
```

Comment:
Could this reference the mediatypes? A table with the following would help:

```
"application/vnd.docker.distribution.manifest.v1+json" manifest
"application/vnd.docker.distribution.manifest.v1+prettyjws" signed manifest
```

Note that "application/json" will also be accepted for schema 1.